projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -115,6 +121,11 @@ export const getSummaryResourceProps = ( state, resourceSlug ) => {
 				ctaLabel: __( 'Add', 'jetpack' ),
 				ctaLink: getJetpackCloudUrl( state, 'settings' ),
 			};
+		case 'vaultpress-backup':
+		case 'vaultpress-for-woocommerce':
+			return {
+				displayName: __( 'VaultPress Backup', 'jetpack' ),
```

Comment:
Exactly. I think that shouldn't be shown in the summary page - since this doesn't really fit as a resource nor as a feature recommendation. But I can ask the team to hear their opinion on that.